docs/source/ops.rst

```diff
@@ -0,0 +1,12 @@
+Modules
+========
+
+.. currentmodule:: pypose
+.. autosummary::
+   :toctree: generated
+   :nosignatures:
+
+   :template: autosummary/class-no-inherit.rst
+   basics.bmv
+   basics.MatrixSquareRoot
+   
```

Comment:
remove this file, we don't need a seperated `rst` file. both `bmv` and `msqrt` are in basics.rst